src/Anon.ts

```diff
@@ -27,10 +27,46 @@ function randomUid() {
 };
 
 //CMHM new function to check that what is in fields is proper
-
-
-
-
+//below are fields that are kept
+// the psuedocode fot what I am going to add is this: 
+// if a field, for exmaple "00080060" contains ("CT" or "MRI" or "Xray" or "flouro" or "US" or "ultrasound" in the DICOM language- must check:
+//     "action":"keep"
+// else 
+// //     "action":"remove"
+// OBVIOUSLY I NEED TO CHECK that CT and MRI are not hashed or encoded or entered as Numbers or , or or,or
+// // "00080016": {"action":"keep", "description":"SOP Class UID"},
+// The following are additional fields to check:
+// "00080060": { "description":"Modality"},
```

Comment:
This is listed in the default set of policies at https://github.com/radiopaedia/dicomanon/blob/master/src/DefaultPolicy.ts#L9